hack/functest.sh

```diff
@@ -1,3 +1,12 @@
 #!/bin/bash -e
 
-echo hello world
+source ./cluster/cluster.sh
+
+[ "$CI" == "true" ] && dnf install -y bind-utils
```

Comment:
Please remove this line as https://github.com/kubevirt/project-infra/pull/2504 is merged and includes
`bind-utils`

Reply:
done